.pre-commit-config.yaml

```
@@ -27,7 +27,7 @@ repos:
     hooks:
       - id: python-file-whitespace
         name: "Check Python file whitespace"
-        entry: 'python Tools/patchcheck/reindent.py --nobackup --newline LF'
+        entry: './Tools/patchcheck/reindent.py --nobackup --newline LF'
```

Also works without the `./`, any preference?

```suggestion
        entry: 'Tools/patchcheck/reindent.py --nobackup --newline LF'
```

---

It's cleaner without the `./`